### PR TITLE
Preparing the 1.1.1 release going forward.

### LINKED ISSUE #8: Greydwarf eyes not buyable

### LINKED ISSUE #10: Buy -> sell -> resell gold dupe [game breaking]
Secret Stash easily exploitable.
able to buy from secret stash for super cheap and sell for a way higher price.
image

### LINKED ISSUE #11: map sync issues
Many people are complaining about losing markers after syncing their maps.

### LINKED ISSUE #19: Gambling is way too cheap.

### PR DESCRIPTION
**Pull request by PolyphonyRequiem**
Creating a space to roll up 1.1.1 changes in preparation for the 1.1.1 release.  If you are making changes intended for the 1.1.1 release, please roll them up into this version.

### CODE REVIEW

**Review by tscheer100 — COMMENTED** [review]
looks good.

**Review by tscheer100 — APPROVED** [review]

**Review by tscheer100 — APPROVED** [review]

**Review by tscheer100 — APPROVED** [review]

**Review by tscheer100 — APPROVED** [review]

**Comment by Firoso** [comment]
Leaving a test comment for discord integration

**Comment by Firoso** [comment]
> Leaving a test comment for discord integration

Again

**Comment by Firoso** [comment]
again, testing discord

**Comment by Firoso** [comment]
Leaving a test comment for discord integration

**Comment by Firoso** [comment]
Leaving a test comment for discord integration

**Comment by Firoso** [comment]
What about here?

File: mod_root/BepInEx/plugins/BetterTrader/item_defaults.cfg [hunk 1]
```diff
@@ -21,6 +21,105 @@ to your liking. Modifying EpicLoot items works the same as any other(see EXAMPLE
...
+Serpent Shield (rationale: mid tier with high risk potential)
+n^2.1 n={75} -  where n is adjusted to be between Iron and Silver
+
+
+Chest & Leggings
+n^2 n={40:100} 			- where n is the tier adjusted to a fit a min/max
+						of 40 for tier 0 (e.g. Rags), to 100 for BlackMetal
+
+Helmets
+n^1.9 n={50:100} 		- where n is the tier adjusted to a fit a min/max
```
Comment (PolyphonyRequiem) [inline mod_root/BepInEx/plugins/BetterTrader/item_defaults.cfg:39]: So, why are helmets cheaper when they provide the same armor value?

File: mod_root/BepInEx/plugins/BetterTrader/item_defaults.cfg [hunk 2]
```diff
@@ -21,6 +21,105 @@ to your liking. Modifying EpicLoot items works the same as any other(see EXAMPLE
...
+						of 40 for tier 0 (e.g. Rags), to 100 for BlackMetal
+
+Capes
+n^1.8 n={50,60,90,100} 	- where n is the tier adjusted to a fit a min/max
```
Comment (PolyphonyRequiem) [inline mod_root/BepInEx/plugins/BetterTrader/item_defaults.cfg:43]: given how little protection is provided by capes, I'd be fine lowering the costs even further.  Except wolf.  That seems good.

File: mod_root/BepInEx/plugins/BetterTrader/item_defaults.cfg [hunk 3]
```diff
@@ -21,6 +21,105 @@ to your liking. Modifying EpicLoot items works the same as any other(see EXAMPLE
...
+						of 50 for tier 1 (e.g. Deer Hide), to 100 for Wolf Cape
+						(rationale: Wolf cape has frost resistance by default
+					 	and is the second most durable cape in the game, if
+					 	enchanted to legedary with maximal affixes it would be
+					 	exceptionally good)
+
+Weapons
+n^2.2 n={70:100} 		- where n is the tier adjusted to a fit a min/max
```
Comment (PolyphonyRequiem) [inline mod_root/BepInEx/plugins/BetterTrader/item_defaults.cfg:51]: Given the serious difference in power between tiers, I feel like this price range is far too tight and too high.

File: mod_root/BepInEx/plugins/BetterTrader/item_defaults.cfg [hunk 4]
```diff
@@ -21,6 +21,105 @@ to your liking. Modifying EpicLoot items works the same as any other(see EXAMPLE
...
+n^2.2 n={85}			- where n is the tier adjusted to a fit a min/max
+						of 85 for tier 3.5 (e.g. Bronze, 80 Iron, 90 Silver),
+						to 100 for BlackMetal
+						(rationale: its a utility weapon and should be tiered
+						to be slightly over the value of a single target weapon
+						of the same material tier as they also require special
+						materials)
+
+FireSword
+n^2.2 n={120}			- where n is the tier adjusted to a fit a min/max
```
Comment (PolyphonyRequiem) [inline mod_root/BepInEx/plugins/BetterTrader/item_defaults.cfg:81]: think I removed this from the trader.

File: mod_root/BepInEx/plugins/BetterTrader/item_defaults.cfg [hunk 5]
```diff
@@ -21,6 +21,105 @@ to your liking. Modifying EpicLoot items works the same as any other(see EXAMPLE
...
+						of 2 for tier 0 (e.g. Necks, tier 1 being boar, tier 2 deer,
+						etc.), to 18 for Yagaluth
+
+Serpent-Trophy
+n^2.4 n = 10.5			- where n is the tier adjusted to a fit a min/max
+						of 2 for tier 0 (e.g. Necks, tier 1 being boar, tier 2 deer,
+						etc.), to 18 for Yagaluth
+						(Same as all other serpent related items, high risk potential)
+
+n^2.4 n = 17.5			- where n is the tier adjusted to a fit a min/max
```
Comment (PolyphonyRequiem) [inline mod_root/BepInEx/plugins/BetterTrader/item_defaults.cfg:99]: what's this o ne referring to?

**Review by tscheer100 — APPROVED** [review]

**Review by tscheer100 — APPROVED** [review]

**Review by tscheer100 — APPROVED** [review]
semicolon, semicolon, semicolon

**Review by tscheer100 — APPROVED** [review]
#AllPathsMatter

**Review by tscheer100 — APPROVED** [review]
rm -rf /

**Review by tscheer100 — APPROVED** [review]

**Review by tscheer100 — APPROVED** [review]

File: mod_root/BepInEx/plugins/BetterTrader/item_defaults.cfg (hunk 1 above)
Comment (maxarady) [reply mod_root/BepInEx/plugins/BetterTrader/item_defaults.cfg:39]: Good point, we can make it match thearmour

File: mod_root/BepInEx/plugins/BetterTrader/item_defaults.cfg (hunk 2 above)
Comment (maxarady) [reply mod_root/BepInEx/plugins/BetterTrader/item_defaults.cfg:43]: Okay, can do

**Review by tscheer100 — APPROVED** [review]

File: mod_root/BepInEx/plugins/BetterTrader/item_defaults.cfg (hunk 4 above)
Comment (maxarady) [reply mod_root/BepInEx/plugins/BetterTrader/item_defaults.cfg:81]: Okay good

**Review by tscheer100 — APPROVED** [review]

File: mod_root/BepInEx/plugins/BetterTrader/item_defaults.cfg (hunk 5 above)
Comment (maxarady) [reply mod_root/BepInEx/plugins/BetterTrader/item_defaults.cfg:99]: hmm?

**Review by tscheer100 — APPROVED** [review]

File: mod_root/BepInEx/plugins/BetterTrader/item_defaults.cfg (hunk 3 above)
Comment (maxarady) [reply mod_root/BepInEx/plugins/BetterTrader/item_defaults.cfg:51]: Okay, widening the gaps is easy enough, as for the price range I feel like its fine as is though, can discuss with the others?

**Pull request merged**